composer.json

```diff
@@ -25,8 +25,12 @@
         "ext-mbstring": "*",
         "yiisoft/strings": "dev-master",
         "yiisoft/arrays": "dev-master",
+        "yiisoft/network-utilities": "dev-master",
         "ext-intl": "*"
     },
+    "suggest": {
```

Comment:
If the package is required, there's no sense in suggesting it.

Reply:
removed